app/assets/javascripts/editor.js

```diff
@@ -65,9 +90,10 @@ $E = {
   italic: function() {
     $E.wrap('_','_')
   },
-  link: function(uri) {
-    uri = uri || prompt('Enter a URL')
-    $E.wrap('[',']('+uri+')')
+  link: function() {
+    uri = prompt('Enter a URL');
+    if (uri == null) { uri = ""; }
+    $E.wrap('[', '](' + uri + ')');
   },
```

Comment:
I added a check for `null` values here. Before, if the user didn't fill out the prompt, `()[null]` would appear in the comment form.